tools/genesis-state-gen/main_test.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/state"
+	ethpb "github.com/prysmaticlabs/prysm/proto/eth/v1alpha1"
+	"github.com/prysmaticlabs/prysm/shared/params"
+	"github.com/prysmaticlabs/prysm/shared/trieutil"
+)
+
+func TestGenerateGenesisState(t *testing.T) {
+	numValidators := 191
+	privKeys, pubKeys, err := deterministicallyGenerateKeys(numValidators)
```

Comment:
anyway to check the keys generated are from the specified curve order ?

Reply:
Not at the moment, unfortunately - it's quite tricky